spec/licensee/matchers/dice_matcher_spec.rb

```diff
@@ -22,12 +22,12 @@
 
   it 'sorts licenses by similarity' do
     expect(subject.licenses_by_similiarity[0]).to eql([gpl, 100.0])
-    expect(subject.licenses_by_similiarity[1]).to eql([agpl, 95.73361082206036])
+    expect(subject.licenses_by_similiarity[1]).to eql([agpl, 86.66032027067445])
```

Comment:
Do we know why this is changing?

Reply:
Yes, GPL and AGPL are long licenses, so they are getting hit hard by dissimilarity scaling with size mentioned at https://github.com/benbalter/licensee/pull/242#discussion_r153684000